components/About/Using.js

```diff
@@ -0,0 +1,54 @@
+// @author Charles Liu
+
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Image from 'next/image'
+
+import styles from '../../styles/Using.module.css'
+
+function Using() {
```

Comment:
is there a more verbose naming then "Using"?

Reply:
The banner is reusable. I'll try to make the content reusable too, but it might be kind of scuffed. Btw, the word verbose actually has a negative connotation. I think you mean "descriptive." Generally, we want to avoid being verbose

Reply:
wdym by the negative connotation with verbose? i would love to learn more

Reply:
it's synonymous to wordy.
![image](https://user-images.githubusercontent.com/65472533/108423284-ad21ab80-7205-11eb-957e-9350cf4d71a0.png)